persistence/pg.go

```diff
@@ -131,6 +139,29 @@ func (p *PostgresPersister) Get(ctx context.Context, hash lntypes.Hash) (*Invoic
 	return invoice, htlcs, nil
 }
 
+func (p *PostgresPersister) GetInvoices(ctx context.Context,
+	maxInvoicesCount, sequenceOffset int) ([]*Invoice, error) {
+
+	var dbInvoices []*dbInvoice
+	err := p.conn.RunInTransaction(ctx, func(tx *pg.Tx) error {
```

Comment:
what is the reasoning to run a single select in a transaction?

Reply:
I will merge for now, but if you have any other concerns about this PR, feel free to comment. I will be more than happy to address them!